src/utils/constants.js

```diff
@@ -12,3 +12,5 @@ export const QUERY_TEMPLATE = '(sortby "%{query.query}")';
 export const INITIAL_RESULT_COUNT = 100;
 
 export const RESULT_COUNT_INCREMENT = 100;
+
+export const DEFAULT_JOB_PROFILE_ID = '6f7f3cd7-9f24-42eb-ae91-91af1cd54d0a';
```

Comment:
I'm not sure, that this is a safe way to track the default profile. Please, could you clarify why we can be sure that it will not be changed on the backend side?

Reply:
We clarified this approach during the grooming of this story.

Reply:
I think the grooming session don't cover all needed details and it's more about overview without diving into detailed implementation. I believe this solution is not reliable enough - UI will not work property if id is changed.  I think we should use an extra field or the user information (we should check if we can trust it) to handle this case.

Reply:
As I pointed above, the id approach is confirmed to bo okay by backend folks. We may be sure that it would stay the same.

Reply:
The UI shouldn't save information about ids of job profiles and mapping profiles. This information should be requested from the backend side. As I mentioned above, the special field or other solution should be added to handle this case.

Reply:
Sorry, but I agree with current solution. If you think that the other kind of solution is needed please contact backed folks. I am doing as part of this PR the things which we agreed upon on grooming. Since you have not stated the thought which you are stating now during that meeting I suggest proceeding with current approach.

Reply:
@VictorSoroka1 Please, take a look at my messages, there are at least 2 reliable solutions

Reply:
I already did but do not have such fields right now, the only one which we have now is a  name which can be really changed at any time. Moreover, this approach is not confirmed by backend folks.

Reply:
Agreed that this temporal solution until the [UIDEXP-128](https://issues.folio.org/browse/UIDEXP-128) is done.